framework/test/mocha/unit/modules/chain/helpers/config.js

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const config = require('../../../../../../src/modules/chain/helpers/config');
+const {
+	ZSchema,
+} = require('../../../../../../src/controller/helpers/validator');
+
+describe('helpers/config', () => {
+	const packageJsonStub = {
+		version: '1.5.0',
+		lisk: {
+			protocolVersion: '1.0',
+		},
+	};
+
+	const env = Object.assign({}, process.env);
+	const baseArgs = [
+		'/Users/michiel.mulders/.nvm/versions/node/v10.15.3/bin/node',
```

Comment:
What if we run these tests not on your computer :D

Reply:
Owh damn that slipped through :D